tensorflow/core/kernels/dml_cross_op.cc

```diff
@@ -105,6 +105,18 @@ class DmlCrossKernel : public DmlKernel {
     // [y, z, x] -> [z, x, y]
     auto vector4 = dml::GatherElements(vector3, indices, 3);
 
+    const auto dtype = ctx->GetOutputDataType(0);
+    const bool is_low_precision_int = dtype == DT_UINT16 || dtype == DT_INT16 ||
+                                      dtype == DT_UINT8 || dtype == DT_INT8;
+
+    // DML doesn't support less than 32bits precision for multiply and subtract
```

Comment:
Interesting, usage of int8 for multiply. I wonder how TF handles overflow. Do you know if it just wraps?

Reply:
I don't know the behavior in TensorFlow, but at least it's consistent. It's unfortunate that the behavior between native vs emulated lower precision support isn't consistent in our case :/